src/cljs/athens/subs.cljs

```diff
@@ -143,3 +143,12 @@
       (:db/remote-graph-conf db)
       {})))
 
+
+;; Appearance Preferences
```

Comment:
Could begin putting some re-frame constructs like subs and events inside the view namespace, e.g. `src/cljs/athens/views/appearance_settings/core.cljs` in this case.

@neotyk do you have preference on colocating re-frame stuff vs keeping in separate events, i.e. all events in `events.cljs`, all subs in `subs.cljs`